src/Sculpin/Core/Tests/Permalink/SourcePermalinkFactoryTest.php

```diff
@@ -52,8 +52,8 @@ public function provideCreateData()
                 ':basename.html',
                 static::makeTestSource('about.md'),
                 new Permalink(
-                    'about.html',
-                    '/about.html'
+                    'about.html/index.html',
```

Comment:
This might introduce a backward compatibility issue for people using relative filenames from within their source files.